components/raftstore-v2/src/operation/ready/mod.rs

```diff
@@ -71,13 +71,33 @@ use crate::{
     worker::tablet,
 };
 
-const PAUSE_FOR_REPLAY_GAP: u64 = 128;
+const PAUSE_FOR_REPLAY_GAP: u64 = 0;
```

Comment:
This need to be discussed.

Reply:
I think it's OK to be 0. It's probably not going to have big impact on start time.